docs/change-log.md

```diff
@@ -2,6 +2,14 @@
 
 This document provides an overview of the changes we are making over time. Feel free to reach out to us in case of any specific questions.
 
+## Product Update April 21st, 2023
+
+- **New:** 
+- **New:** Introduced [Versioning of Shared Values and overrides](https://docs.humanitec.com/using-humanitec/work-with-shared-values-and-secrets/versioning-of-values-and-secrets).
+- **Improved:** Added a "learn more" link to error messages for past deployments to make troubleshooting issues easier.
+- **Improved:** It is now possible to review usage of Humanitec's default resources in the [Resource Management screen](https://docs.humanitec.com/reference/user-interface/resource-management).
+- **Improved:** Deployment diff now shows changes even if the application hasn't been deployed.
```

Comment:
Just for my understanding: Previously when hitting "deploy" on a new application the deployment diff was empty? (even if I made changes: added worksloads etc)

Reply:
That was my understanding. Here's the ticket: https://humanitec.atlassian.net/browse/WAL-6835